app/src/main/java/com/vitorpamplona/amethyst/ui/dal/ChannelFeedFilter.kt

```diff
@@ -16,6 +16,10 @@ object ChannelFeedFilter: FeedFilter<Note>() {
 
   // returns the last Note of each user.
   override fun feed(): List<Note> {
-    return channel.notes?.values?.filter { account.isAcceptable(it) }?.sortedBy { it.createdAt() }?.reversed() ?: emptyList()
+    return channel.notes
+      ?.values
+      ?.filter { account.isAcceptable(it) }
+      ?.sortedBy { it.createdAt() }
+      ?.reversed() ?: emptyList()
```

Comment:
`val notes = ConcurrentHashMap<HexKey, Note>()`

It's non-nullable, these safety checks can all be removed.